webroot/admin/group-mgmt.php

```diff
@@ -18,12 +18,18 @@
         case "req":
             if ($_POST["action"] == "Approve") {
                 // approve group
-                $group = $form_user->getPIGroup();
-                $group->approveGroup($OPERATOR);
+                $group_type = $_POST["group_type"];
+                $group_name = $_POST["group_name"];
+                $group_uid = $group_type . "_" . $group_name;
+                $group = new UnityGroup($group_uid, $LDAP, $SQL, $MAILER, $REDIS, $WEBHOOK);
+                $group->approveGroup($_POST["uid"], $OPERATOR);
```

Comment:
When trying to approve any group, I get this php error. In my case the requestor of the group is `user1@org1.test` and the approver is also `user1@org1.test`.

```

Notice: Trying to access array offset on value of type bool in /var/www/unity-web-portal/resources/lib/UnitySQL.php on line 649

Fatal error: Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'user' cannot be null in /var/www/unity-web-portal/resources/lib/UnitySQL.php:524 Stack trace: #0 /var/www/unity-web-portal/resources/lib/UnitySQL.php(524): PDOStatement->execute() #1 /var/www/unity-web-portal/resources/lib/UnitySQL.php(766): UnityWebPortal\lib\UnitySQL->assignRole() #2 /var/www/unity-web-portal/resources/lib/UnityGroup.php(538): UnityWebPortal\lib\UnitySQL->assignSuperRole() #3 /var/www/unity-web-portal/resources/lib/UnityGroup.php(172): UnityWebPortal\lib\UnityGroup->init() #4 /var/www/unity-web-portal/webroot/admin/group-mgmt.php(25): UnityWebPortal\lib\UnityGroup->approveGroup() #5 {main} thrown in /var/www/unity-web-portal/resources/lib/UnitySQL.php on line 524
```